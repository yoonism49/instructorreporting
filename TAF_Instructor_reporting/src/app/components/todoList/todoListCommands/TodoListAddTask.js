// @flow weak

import React        from 'react';

const TodoListAddTask = () => (
  <a
    className="btn btn-success btn-sm pull-left"
    href="#">
    Add New Tasks
  </a>
);

export default TodoListAddTask;
