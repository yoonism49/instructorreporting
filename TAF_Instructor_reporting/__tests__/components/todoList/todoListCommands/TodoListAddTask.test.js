'use strict';

import React              from 'react';
import TodoListAddTask    from '../../../../src/app/components/todoList/todoListCommands/TodoListAddTask';
import renderer           from 'react-test-renderer'; // needed both for snpashot testing but also to prevent errors from enzyme

describe('TodoListAddTask component', () => {
  it('renders as expected', () => {
    const component = renderer.create(
      <div>
        <TodoListAddTask />
      </div>
    ).toJSON();
    expect(component).toMatchSnapshot();
  });
});
