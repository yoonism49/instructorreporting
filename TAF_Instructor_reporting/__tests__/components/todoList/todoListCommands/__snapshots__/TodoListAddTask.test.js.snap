// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoListAddTask component renders as expected 1`] = `
<div>
  <a
    className="btn btn-success btn-sm pull-left"
    href="#"
  >
    Add New Tasks
  </a>
</div>
`;
